### PR TITLE
Add separator for VTM-offline-maps

### PR DESCRIPTION
**Pull request by murggel**
## Description

Consider `AbstractMapsforgeVTMOfflineTileProvider` and `MapsforgeVTMMultiOfflineTileProvider` also for separated offline-maps

### CODE REVIEW

File: main/src/main/java/cgeo/geocaching/unifiedmap/tileproviders/TileProviderFactory.java [hunk 1]
```diff
@@ -57,7 +57,10 @@ public static void addMapviewMenuItems(final Activity activity, final PopupMenu
             }
             if (!hide) {
                 final int id = tileProvider.getNumericalId();
-                parentMenu.add(tileProvider instanceof AbstractMapsforgeOfflineTileProvider ? R.id.menu_group_map_sources_offline : R.id.menu_group_map_sources_online, id, i, tileProvider.getTileProviderName()).setCheckable(true).setChecked(id == currentTileProvider);
+                final boolean isOfflineMap = (tileProvider instanceof AbstractMapsforgeOfflineTileProvider)
+                        || (tileProvider instanceof AbstractMapsforgeVTMOfflineTileProvider)
+                        || (tileProvider instanceof MapsforgeVTMMultiOfflineTileProvider);
```
Comment (moving-bits) [inline main/src/main/java/cgeo/geocaching/unifiedmap/tileproviders/TileProviderFactory.java:62]: the last part (`tileprovider instanceof MapsforgeVTMMultiOfflineTileProvider`) is not required, as a `MapsforgeVTMMultiOfflineTileProvider` is a subclass of `MapsforgeVTMOfflineTileProvider`

**Review by moving-bits — CHANGES_REQUESTED** [review]
Thanks for diving into this issue, good idea!

PR generally LGTM, just one small finding, see comment.

File: main/src/main/java/cgeo/geocaching/unifiedmap/tileproviders/TileProviderFactory.java (hunk 1 above)
Comment (murggel) [reply main/src/main/java/cgeo/geocaching/unifiedmap/tileproviders/TileProviderFactory.java]: Removed

**Comment by murggel** [comment]
Is it on purpose, that the `OpenStreetMap` has a different name `OSM `in unifiedMap?

**Comment by moving-bits** [comment]
> Is it on purpose, that the `OpenStreetMap` has a different name `OSM `in unifiedMap?

That's probably just been a shortcut back then when I started implementing UnifiedMap. Should better be the same as in the "old" maps, and probably preferably the long form to get the connection to the related web services.

**Comment by murggel** [comment]
> > Is it on purpose, that the `OpenStreetMap` has a different name `OSM `in unifiedMap?
> 
> That's probably just been a shortcut back then when I started implementing UnifiedMap. Should better be the same as in the "old" maps, and probably preferably the long form to get the connection to the related web services.


I will provide a PR for that...

**Review by moving-bits — APPROVED** [review]
LGTM, thanks for the contribution

**Pull request merged**